### ISSUE TITLE
Hardware Trigger

### ISSUE THREAD

**Issue by m91moreira** [ros-drivers/camera1394#46]
Hi!

I've the firefly mv FMVU-03MTC and I'm already able to publish the image using the camera1394_node. Now I would like to initialize the node with the hardware trigger option. I can't find any examples/documentation of that, can you help me?

**Comment by mhkabir** [comment]
@jack-oquin I have hardware trigger working, but I see 100% CPU usage when using it. I would be happy to document the feature once I get it satisfactorily working.
Also, if the driver is running and I unplug the camera, the CPU usage goes to 100%
What might be the issue here?

**Comment by jack-oquin** [comment]
@mhkabir: thanks for the report, please forgive the slow response. I have trouble testing hardware triggering with the test cameras I have available. So, your experience should be helpful.

What device are you using? A similar Firefly model?

What trigger parameters are you passing to the camera1394 driver?

**Comment by mhkabir** [comment]
@jack-oquin Yes,  I'm using a similar firefly model as you will have seen on my issue report. 
Move discussion over to #48
